grin/src/Transformations/ExtendedSyntax/Conversion.hs

```diff
@@ -248,6 +250,7 @@ instance Convertible New.Exp Exp where
   convert (New.SReturn val)        = SReturn (convert val)
   convert (New.SBlock exp)         = SBlock (convert exp)
   convert (New.Alt cpat exp)       = Alt (convert cpat) (convert exp)
+  convert (New.NAlt cpat _ exp)    = Alt (convert cpat) (convert exp)
```

Comment:
If the name from the named alt are referred in the body of the Alt, this conversion is not valid. Please make a TODO comment there for the future refactoring, or just simply raise a runtime error.

Reply:
That is a valid coven. I think a TODO will suffice for now.

Reply:
Added a TODO